### PR TITLE
Earth age map format

### PR DESCRIPTION
**Pull request by Esteban82**
See #104 for details.

With this PR I:

1.  use the jpg version for earth_age
2. delete the png figures that are not being used anymore.

#### Doubt
Should I also delete the scripts for `GMT_earth_mask.sh` and `GMT_earth_age.sh`?

### CODE REVIEW

**Review by yvonnefroehlich — APPROVED** [review]

**Comment by Esteban82** [comment]
I would need some else approval. Yvone doesn't have write access it seems so it doesn't count.

**Review by PaulWessel — APPROVED** [review]

**Pull request merged**
